tests/test_io.py

```diff
@@ -88,10 +88,27 @@ def test_load_population():
     population = io.load_population(POP_PATH)
     nt.ok_(len(population.neurons) == 5)
     names = np.array([os.path.basename(n.name) for n in population.neurons])
-    
+
     L = glob.glob(POP_PATH + '/*')
     population1 = io.load_population(L)
     nt.ok_(len(population.neurons) == 5)
     names1 = np.array([os.path.basename(n.name) for n in population1.neurons])
     assert_array_equal(names, names1)
     nt.ok_(population.neurons[0].is_equal(population1.neurons[0]))
+
+
+def test_tree_type():
+    tree_types = {5: 'soma',
+                  6: 'axon',
+                  7: 'basal',
+                  8: 'apical'}
+
+    neuron = io.load_neuron(os.path.join(DATA_PATH, 'basic_exotic_section_types.swc'),
+                               soma_type=5,
```

Comment:
It's a bit confusing that the soma type is defined both by the tree_types dictionary and the soma_type argument